docker_cli/cli/setup_commands.py

```diff
@@ -0,0 +1,237 @@
+
+    setup_cuda_dev_parser = cuda_subparsers.add_parser(
+        "dev",
+        parents=[setup_parse, no_cache_parse],
+        help="Set up the CUDA dev image.",
+        formatter_class=help_formatter,
+    )
+    add_tag_argument(parser=setup_cuda_dev_parser, default="cuda-dev")
+
+    setup_conda_parser = setup_subparsers.add_parser(
+        "conda",
+        help="Set up a conda environment image. Designate dev or runtime.",
+        formatter_class=help_formatter,
+    )
+
+    conda_subparsers = setup_conda_parser.add_subparsers(
+        dest="conda_subcommand", required=True
+    )
+    setup_conda_runtime_parser = conda_subparsers.add_parser(
+        "runtime",
+        parents=[setup_parse, no_cache_parse],
+        help="Set up the runtime conda environment image",
+        formatter_class=help_formatter,
+    )
+    setup_conda_runtime_parser.add_argument(
+        "--env-file",
+        default=run_reqs_path_str,
+        type=str,
+        help="The location of the runtime requirements file. Can be a pip-style "
+        "requirements.txt file, a conda-style environment.yml file, or a lockfile.",
+    )
+    add_tag_argument(parser=setup_conda_runtime_parser, default="conda-runtime")
+
+    setup_conda_dev_parser = conda_subparsers.add_parser(
+        "dev",
+        parents=[setup_parse, no_cache_parse],
+        help="Set up the dev conda environment image",
+        formatter_class=help_formatter,
+    )
+    setup_conda_dev_parser.add_argument(
+        "--env-file",
+        default=dev_reqs_path_str,
+        type=str,
+        help="The location of the dev requirements file. Can be a pip-style "
+        "requirements.txt file, a conda-style environment.yml file, or a lockfile.",
+    )
+    add_tag_argument(parser=setup_conda_dev_parser, default="conda-dev")
+
+    # This command has been commented out due to buggy implementation, but may
+    # later be recovered once a better implementation has been found.
+    """setup_conda_add_parser = conda_subparsers.add_parser(
+        "add",
+        parents=[setup_parse, no_cache_parse],
+        help="Set up the runtime conda environment image",
+        formatter_class=help_formatter,
+    )
+    setup_conda_add_parser.add_argument(
+        "packages",
+        nargs="+",
+        help="A list of conda packages to add to the environment.",
+    )
+    setup_conda_add_parser.add_argument(
+        "--channels",
+        "-c",
+        nargs="+",
+        help="A list of channels to look for conda packages in.",
+    )
+    add_tag_argument(parser=setup_conda_add_parser, default="conda-pkgs")"""
+
+    return
+
+
+def run_setup(args: argparse.Namespace) -> None:
+    setup_subcommand: str = args.setup_subcommand
+    del args.setup_subcommand
+    if setup_subcommand == "all":
+        images = setup_all(**vars(args))
+        print("IMAGES GENERATED:")
+        for image_tag in images:
+            print(textwrap.indent(image_tag, "\t"))
```

Comment:
This doesn't belong here.

If you want, you could add a `--verbose` option that gets passed to the `setup_all()` function, and then print this info from inside that function if the user requested verbose output.

```suggestion
```